Makefile

```diff
@@ -32,34 +32,41 @@ CONFDIR=	${PREFIX}/etc
 LIBEXECDIR=	${PREFIX}/libexec
 SHAREDIR=	${PREFIX}/share
 
-DIRS=		SCRIPTSDIR
-
 SCRIPTS=	dynamic_motd
 SCRIPTSDIR=	${LIBEXECDIR}
 
-FILESGROUPS=	DOCS EXAMPLES
-
 DOCS=		CONTRIBUTING.md README.md
 DOCSDIR=	${SHAREDIR}/doc/dynamic_motd
 
 EXAMPLES=	examples/motd.subr examples/rc.motd
 EXAMPLESDIR=	${SHAREDIR}/examples/dynamic_motd
 
-CONFGROUPS=	CONFETC CONFRCD
-
 CONFETC=	rc.motd.sample
 
 CONFRCD=	rc.d/dynamic_motd
 CONFRCDDIR=	${CONFDIR}/rc.d
-CONFRCDMODE=	${BINMODE}
 
 CLEANFILES=	rc.d/dynamic_motd
 PREFIX_SUB=	-e 's,@@PREFIX@@,${PREFIX},g'
 
+INSTALL_DATA=	install -m 0644
+INSTALL_SCRIPT=	install -m 0555
+MKDIR=		mkdir -p
+
+all: rc.d/dynamic_motd
+
 rc.d/dynamic_motd: rc.d/dynamic_motd.in
 	sed ${PREFIX_SUB} ${.ALLSRC} >${.TARGET}
 
-beforeinstall: installdirs
-afterinstall: installconfig
+install: installdirs
```

Comment:
It looks like there might be a missing dependency on `rc.d/dynamic_motd` here. If you don't mind, I can update your patch later this afternoon.

Reply:
By all means, although I don't understand what the error is?  

This is where `rc.d/dynamic_motd` gets installed:
```
	${INSTALL_SCRIPT} ${CONFRCD}  ${DESTDIR}${CONFRCDDIR}
```
I tried my patch in a FreeBSD ports build and there were no plist problems encountered.

Reply:
I'm puzzled too. From [CI](https://github.com/sstallion/dynamic_motd/actions/runs/4720348686/jobs/8375585826?pr=2#step:3:1073):
```
 + make install
  mkdir -p /usr/local/etc/rc.d
  mkdir -p /usr/local/etc
  mkdir -p /usr/local/libexec
  mkdir -p /usr/local/share/doc/dynamic_motd
  mkdir -p /usr/local/share/examples/dynamic_motd
  install -m 0555 rc.d/dynamic_motd  /usr/local/etc/rc.d
  install: rc.d/dynamic_motd: No such file or directory
  *** Error code 71
```
It looks to me that dependency resolution for `rc.d/dynamic_motd` is broken somehow. The easiest fix would be to add it as a direct dependency.

It could be a difference in how make is being called in CI (ie. calling make directly rather than through ports). I wouldn't expect it, but it could be a minor difference in behavior between 13.0-RELEASE and newer versions too. Either way, I can work through it at lunch today and get your changes merged.

Thanks again!

Reply:
Ah.  I see what the problem is.  I was assuming that people would always have run `make all` before separately trying to run `make install`.  The update in 1600c89 should fix the problem.